eng/deploy.yaml

```diff
@@ -1,14 +1,60 @@
-parameters:
-  Subscription: ''
-  ServiceFabricConnection: ''
-  PublishProfile: ''
-  DotNetStatusAppName: ''
-  DeploymentEnvironment: ''
-  VariableGroup: ''
-  MaestroPublishEndpoint: ''
-  MaestroTestEndpoint: ''
-  DotNetStatusEndpoint: ''
-  StatusVariableGroup: ''
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: Deploy
```

Comment:
Why is this resource named "Deploy". It is the build, not the deploy artifacts.

Reply:
Changed to Build.